### PR TITLE
Add scripts and repository role for setup and teardown

### PR DESCRIPTION
**Pull request by erzetpe**

### CODE REVIEW

File: core/src/epicli/data/aws/defaults/infrastructure/virtual-machine.yml [hunk 1]
```diff
@@ -223,14 +223,14 @@ specification:
        destination_port_range: "22"
        source_address_prefix: "0.0.0.0/0"
        destination_address_prefix: "0.0.0.0/0"
-     - name: node_exporter
-       description: Allow node_exporter traffic
+     - name: repository
```
Comment (seriva) [inline core/src/epicli/data/aws/defaults/infrastructure/virtual-machine.yml]: Seems you replaced the nope_exporter entry with this one. Shouldn't there be both?

File: core/src/epicli/cli/engine/ansible/AnsibleRunner.py [hunk 2]
```diff
@@ -47,11 +47,20 @@ def run(self):
...
+
+        if repository_setup_play_result != 0:
+            return
+
         common_play_result = self.ansible_command.run_playbook_with_retries(inventory=inventory_path,
                                                                             playbook_path=os.path.join(
                                                                                 get_ansible_path(
                                                                                     self.cluster_model.specification.name),
-                                                                                "common.yml"), retries=5)
+                                                                                "common.yml"), retries=1)
```
Comment (seriva) [inline core/src/epicli/cli/engine/ansible/AnsibleRunner.py]: Is there a reason we only want one retry for the other playbooks now?

File: core/src/epicli/cli/engine/ansible/AnsibleRunner.py [hunk 3]
```diff
@@ -47,11 +47,20 @@ def run(self):
 
         self.ansible_vars_generator.run()
 
+        repository_setup_play_result = self.ansible_command.run_playbook_with_retries(inventory=inventory_path,
```
Comment (seriva) [inline core/src/epicli/cli/engine/ansible/AnsibleRunner.py]: I think you need to check the offline flag here.  Dont we only want to execute this when we are in online mode?

File: core/src/epicli/data/aws/defaults/infrastructure/virtual-machine.yml (hunk 1 above)
Comment (erzetpe) [reply core/src/epicli/data/aws/defaults/infrastructure/virtual-machine.yml]: Need to fix.

Comment (erzetpe) [reply core/src/epicli/data/aws/defaults/infrastructure/virtual-machine.yml]: Fixed.

**Pull request merged**